### ISSUE TITLE
Streaming HuBERT

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#61]

**Comment by StrongTanisha** [comment]
Experiment ID: 03038eeb-7106-4a5e-95ec-fc91ae703751

Has checkpointing in-built

**Comment by StrongTanisha** [comment]
Calvin - still to write up the readme

ISC demos - instructions for set up
Take necessary stuff from other repo

**Comment by StrongTanisha** [comment]
* first training run
* extract a layer from that
* get these labels
* then train again
* 2 files that we have
* create ehisc file
* also have to modify the training config file that is within fairsec
* so if we clone it, we lose those two
* Read the paper - need to create a data processing thing (doing on the train)
* 10 is not how the paper specified it
* K means clustering - X number of different chards
* Those shards are similar - didn’t read the paper properly - it’s actually 100
* Calvin to redo on 100
* Currently working on pre-training step (first stage of pre-training step)
* Replicated the paper configuration and currently cycling through
* Deliverable being developed. Once it’s passed the first iterations, will be as smooth as possible for developers to get on and have their workflow replicated
